CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributor Guideline
+
+
+## Commit
+
+> One golden rule: follow [the commit format][commit-format]
+
+We use [standard-version][standard-version] to automatically generate [CHANGELOG.md][CHANGELOG.md] from the `git commit` history. Please **do** follow [the commit format][commit-format] before commiting.
+
+
+## PR
+
+Simple rules:
+
+* Do rebase before merging.
```

Comment:
@tomchentw Sorry for the silly question. Could you please clarify to me what you mean here? You mean rebase master into the branch?  

Does this mean we don't merge master into the branch?